iff -ur pinentry-0.7.2.orig/doc/pinentry.info pinentry-0.7.2/doc/pinentry.info
--- pinentry-0.7.2.orig/doc/pinentry.info       2005-01-27 02:06:25.000000000 -0600
+++ pinentry-0.7.2/doc/pinentry.info    2006-05-11 12:27:06.000000000 -0500
@@ -3,7 +3,7 @@

 INFO-DIR-SECTION GNU Utilities
 START-INFO-DIR-ENTRY
-* pinentry::               Ask securely for a passphrase or PIN.
+* pinentry: (pinentry).               Ask securely for a passphrase or PIN.
 END-INFO-DIR-ENTRY
    This file documents the use and the internals of the PINENTRY.

